moveit_commander/src/moveit_commander/move_group.py

```diff
@@ -126,6 +126,26 @@ def set_start_state_to_current_state(self):
         self._g.set_start_state_to_current_state()
 
     def set_start_state(self, msg):
+        """
+        Specify a start state for the group.
+
+        Parameters
+        ----------
+        msg : moveit_msgs/RobotState
+
+        Examples
+        --------
+        >>> from moveit_msgs.msg import RobotState
+        >>> from sensor_msgs.msg import JointState
+        >>> joint_state = JointState()
+        >>> joint_state.header = Header()
+        >>> joint_state.header.stamp = rospy.Time.now()
+        >>> joint_state.name = joint_name
+        >>> joint_state.position = joint_position
```

Comment:
I like the example!

`joint_name` and `joint_position` are actually lists of values.
Could you fill them in with fictional values to set e.g. `['joint_a', 'joint_b']` instead of the non-existing variables?